tests/unit/test_taxonomy.py

```diff
@@ -107,31 +102,20 @@ def test_find_deepest_nodes(
     [
         (TaxonomyType.brand.name, "carrefour-bio", "Carrefour Bio"),
         (TaxonomyType.brand.name, "unknown-brand", None),
-        (TaxonomyType.label.name, "eu-organic", "en:eu-organic"),
-        (TaxonomyType.label.name, "unknown-label", None),
-    ],
-)
-def test_match_unprefixed_value(taxonomy_type, value, expected):
-    assert match_unprefixed_value(value, taxonomy_type) == expected
-
-
-@pytest.mark.parametrize(
-    "taxonomy_type,value,expected",
-    [
-        (TaxonomyType.brand.name, "carrefour-bio", "Carrefour Bio"),
-        (TaxonomyType.brand.name, "unknown-brand", None),
-        (TaxonomyType.label.name, "eu-organic", "en:eu-organic"),
+        (TaxonomyType.label.name, "fr:bio-europeen", "en:eu-organic"),
         (
             TaxonomyType.label.name,
             "ab-agriculture-biologique",
-            "fr:ab-agriculture-biologique",
+            None,
```

Comment:
Isn't it a downgrade not to allow ab-agriculture-biologique to match taxonomy ?

Reply:
we now only accept language-prefixed values for logo, so no it's expected!